apps/copilot-api/knexfile.js

```diff
@@ -0,0 +1,20 @@
+// knexfile.js
+
+require("dotenv").config();
+
+module.exports = {
+    development: {
+        client: "pg",
+        connection: process.env.DATABASE_URL,
+        migrations: {
+            directory: "./migrations",
+        },
+    },
+    production: {
```

Comment:
I see these two have the same params, is this intended?

Reply:
for now yes